src/ODEmodel.jl

```diff
@@ -50,7 +50,7 @@ end
 
 
 """ Predicts the model given a set of parametrs. """
-function predict(p, g_0::Real, t, nG1::Integer, nG2::Integer, nD1, nD2, retVec = false)
+function predict(p, g_0::Real, t, nG1::Integer, nG2::Integer, nD1, nD2, vec)
```

Comment:
```suggestion
function predict(p, g_0::Real, t, nG1::Integer, nG2::Integer, nD1, nD2, vec = nothing)
```

This will make `vec` default to nothing unless you provide it.

Reply:
Thanks